cumulusci/tasks/robotframework/robot_lint.py

```diff
@@ -0,0 +1,219 @@
+from cumulusci.core.tasks import BaseTask
```

Comment:
Yeah, I was wondering about that. I like the idea of naming the file after the task rather than partly after the task, but you're right that the leading robot_ is redundant, and leaving it off is more consistent with the libdoc task which also leaves it off. 

I'll change it.

Reply:
I kind of like that idea too -- although there's potential to have multiple named task configs that use the same class, and it would be a big refactor to standardize on that across all the tasks. It would help avoid the current problem where we have some task classes named the same as their modules though, which can make importing the module difficult if the package __init__.py imports the class.